dsc/templates/dsc/inc/deviceserver_documentation.html

```diff
@@ -46,16 +47,14 @@ <h4>{{doc.get_documentation_type_display}}</h4>
 {% if readme or readme_link %}
     <div style="word-wrap: break-word;">
         <h4>README</h4>
-        {% if rst %}
-            {{ readme|apply_markup:"restructuredtext" }}
-        { % elif md %}
-            {{readme|apply_markup:"markdown"}}
+        {% if rst or deviceserver.description|find_match:'.*:.*:.*'%}
```

Comment:
Should it be test on readme?:
```suggestion
        {% if rst or readme|find_match:'.*:.*:.*'%}
```